Modules/Core/Transform/include/itkCompositeTransform.hxx

```diff
@@ -446,17 +446,7 @@ template <typename TParametersValueType, unsigned int VDimension>
 auto
 CompositeTransform<TParametersValueType, VDimension>::GetInverseTransform() const -> InverseTransformBasePointer
 {
-  /* This method can't be defined in Superclass because of the call to New() */
-  Pointer inverseTransform = New();
-
-  if (this->GetInverse(inverseTransform))
-  {
-    return inverseTransform.GetPointer();
-  }
-  else
-  {
-    return nullptr;
-  }
+  return Superclass::InvertTransform(*this);
```

Comment:
Looks like the original intention was already to implement `GetInverseTransform()` in its `Superclass`, as I understand from this comment, added by Michael Stauffer, commit d9de55c13c0632cc13fbb0f60f2b8f08eae1de0c (2012), saying `/* This method can't be defined in Superclass because of the call to New() */`